src/cloudcasting/metrics.py

```diff
@@ -89,14 +82,16 @@ def ssim_single(input: SingleArray, target: SingleArray, win_size: int | None =
     ssim_seq = []
     for i_t in range(input.shape[1]):
         _, ssim_array = structural_similarity(
-            input[:, i_t],
-            target[:, i_t],
+            input[:, i_t, :, :],
```

Comment:
Worth adding a comment here to explain what this array slicing operation is trying to do - they're a bit opaque unless you know what all the dimensions represent.